src/platform/nrfconnect/FactoryDataParser.c

```diff
@@ -17,12 +17,32 @@
 
 #include "FactoryDataParser.h"
 
+#include <zcbor_common.h>
 #include <zcbor_decode.h>
+#include <zcbor_encode.h>
 
 #include <ctype.h>
 #include <string.h>
 
-#define MAX_FACTORY_DATA_NESTING_LEVEL 4
+#define zcbor_tstr_put_lit_cast(state, string) zcbor_tstr_encode_ptr(state, (char *) string, sizeof(string) - 1)
+
+#define MAX_FACTORY_DATA_NESTING_LEVEL 2
```

Comment:
I can see that we previously reduced the nesting level because we don't parse the `user_data` field using the factory data parser, but we didn't change that value. We have only two levels in the factory data set, a map and then values.

Reply:
I see, thanks